letsencrypt/display/enhancements.py

```diff
@@ -11,7 +11,7 @@
 logger = logging.getLogger(__name__)
 
 # Define a helper function to avoid verbose code
-util = zope.component.getUtility  # pylint: disable=invalid-name
```

Comment:
Again, may I please ask you to refrain updating code that is irrelevant to the PR?